File: csharp/DVSClient/DVSClient/server/address/format/RestApiAddressFormatEnrichment.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Format
{
    public class RestApiAddressFormatEnrichment
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("transaction_id")]
        public string? TransactionId { get; set; }

        [JsonProperty("result")]
        public RestApiEnrichmentResultAddress? Result { get; set; }

        [JsonProperty("metadata")]
        public RestApiEnrichmentMetadata? Metadata { get; set; }
    }
}

File: typescript/src/server/address/format/restApiAddressFormatEnrichment.ts
import { RestApiResponseError } from "../../restApiResponseError";
import { RestApiEnrichmentMetadata } from "./restApiEnrichmentMetadata";
import { RestApiEnrichmentResultAddress } from "./restApiEnrichmentResultAddress";

export type RestApiAddressFormatEnrichment = {
    error?: RestApiResponseError;
    transaction_id?: string;
    result?: RestApiEnrichmentResultAddress;
    metadata?: RestApiEnrichmentMetadata;
};